samples: nrf9160: modem_shell: bluetooth support (#7876)
nrfconnect/sdk-nrf: samples/nrf9160/modem_shell/README.rst

The LEDs have the following functions:

> LED 3 (nRF9160 DK) / Blue LED (Thingy91):

Reviewer: ```suggestion
LED 3 (nRF9160 DK)/Blue LED (Thingy:91):
```

   west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-non-offloading.conf

BT shell support
================

To build the MoSh sample with Zephyr BT shell command support, use ``-DDTC_OVERLAY_FILE=bt.overlay`` and  ``-DOVERLAY_CONFIG=overlay-bt.conf`` options.

> When running this configuration, BT scanning and advertising can be mabe with ``bt`` command. Compilation for an example:

Author: fixed

> To build the MoSh sample with Zephyr BT shell command support, use ``-DDTC_OVERLAY_FILE=bt.overlay`` and  ``-DOVERLAY_CONFIG=overlay-bt.conf`` options.
> When running this configuration, BT scanning and advertising can be made with ``bt`` command. Compilation for an example:

Reviewer: ```suggestion
To build the MoSh sample with Zephyr BT shell command support, use the :file:`-DDTC_OVERLAY_FILE=bt.overlay` and :file:`-DOVERLAY_CONFIG=overlay-bt.conf` options.
When running this configuration, you can perform BT scanning and advertising using the ``bt`` command.

Compile as follows:
```

.. code-block:: console

   west build -p -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE="bt.overlay" -DOVERLAY_CONFIG="overlay-bt.conf"

> Additionally, NRF52 side of the development kit needs to be programmed as instructed in :ref:`lte_ble_gateway`. Compilation of :ref:`uart_nrf_sw_lpuart`:

Reviewer 2: The `n` in `nRF52` should be lower case.

Author: fixed

> Additionally, nRF52840 side of the development kit needs to be programmed as instructed in :ref:`lte_sensor_gateway`. Compilation of :ref:`bluetooth-hci-lpuart-sample`:

Reviewer: ```suggestion
Additionally, you need to program the nRF52840 DK as instructed in :ref:`lte_sensor_gateway`.

Compile the :ref:`bluetooth-hci-lpuart-sample` as follows:
```

Author: Thanks for the comments. But this one is not quite correct. I'm referring to nRF52840 that is on/part of the nRF9160 DK. I.e. it is not nRF52840 DK.

Reviewer: Thought this might go wrong. But you got the idea how to fix this.

Author: Yes, I think so. All has been fixed now.

.. code-block:: console

   west build -p -b nrf9160dk_nrf52840

> Following example demonstrates how to use MoSh with 2 development kits, where one development kit acts as a broadcaster and another one as an observer.

Reviewer: ```suggestion
The following example demonstrates how to use MoSh with two development kits, where one acts as a broadcaster and the other one as an observer.
```

> DK #1 where MoSh used in a broadcaster (advertising) role:

Reviewer: ```suggestion
DK #1, where MoSh is used in broadcaster (advertising) role:
```

   .. code-block:: console

      mosh:~$ bt init
      Bluetooth initialized
      Settings Loaded
      mosh:~$ bt name mosh-adv
      mosh:~$ bt name
      Bluetooth Local Name: mosh-adv
      mosh:~$ bt advertise scan
      Advertising started

      /* And when done: */
      mosh:~$ bt advertise off
      Advertising stopped
      mosh:~$

> DK #2 where MoSh used in an observer (scanning) role:

Reviewer: ```suggestion
DK #2, where MoSh is used in observer (scanning) role:
```
